BunqSdk/Exception/ApiException.cs

```diff
@@ -1,32 +1,17 @@
-﻿using System.Collections.Generic;
-
+﻿
 namespace Bunq.Sdk.Exception
 {
-    /// <summary>
-    /// Exception triggered by API requests failed on the server side.
-    /// </summary>
     public class ApiException : System.Exception
     {
-        /// <summary>
-        /// Glue to concatenate the error messages.
-        /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
-
         public int ResponseCode { get; private set; }
-        public IList<string> Messages { get; private set; }
+        public string Messages { get; private set; }
```

Comment:
doesn't it become a singular "message"? :D (like in PHP)

Reply:
🙈